### DISCUSSION TITLE
Does Seissol support drawing time step width distribution map?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#791] [Q&A] [answered]
I notice in SeisSol outlog file that give min. time step width, Does Seissol code support drawing time step width distribution map in paraview or in gmsh?
![image](https://user-images.githubusercontent.com/114474894/217511636-7ea50224-c302-4099-ae72-2cb3a97e1b4f.png)

Similar to the velocity field distribution in the figure below, can you get the distribution of time step width on each grid in paraview or gmsh?
![image](https://user-images.githubusercontent.com/114474894/217514292-11d14320-82d2-4a6e-8b1e-eef78813d54d.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
yes, it is outputted in the volume output (variable clustering).

**Reply by buzhiqiuyuebai** [reply]
> yes, it is outputted in the volume output (variable clustering).

Thank you for your help! By the way, is the time unit in ms?

**Reply by Thomas-Ulrich** [reply]
no, it is the cluster id. E.g. cluster n would have dt = 2^n * dt_min (or 2^(n-1), I m not sure)
